roles/common/files/uug_ansible_wrapper.py

```diff
@@ -388,7 +388,10 @@ def on_run_clicked(self, _):
         if not validate_branch():
             invalid_branch(self)
             return
-
+        
+        if not validate_config_settings(self):
+            continue
```

Comment:
So this shouldn't actually be a `continue` statement. That's only valid in loops. Instead, can you replace this with a call to the logging library?
```suggestion
            logging.warn("Non-optimal user branch settings.")
```